controllers/scc.go

```diff
@@ -52,14 +52,16 @@ func (c openshiftSccs) ensureCreated(r *LVMClusterReconciler, ctx context.Contex
 			r.Log.Info("creating SecurityContextConstraint", "SecurityContextConstraint", scc.Name)
 			_, err := r.SecurityClient.SecurityContextConstraints().Create(ctx, scc, metav1.CreateOptions{})
 			if err != nil {
-				return fmt.Errorf("failed to create SCC %q: %v", scc.Name, err)
```

Comment:
Please include the scc name in the error and the log

Reply:
err will already have it, But I have added it again in the logs.